app/assets/javascripts/components/course/course.jsx

```diff
@@ -56,6 +57,17 @@ export const Course = withRouter(createReactClass({
     return `${course_school}/${course_title}`;
   },
 
+  showCourseApprovalForm() {
```

Comment:
This would benefit from a comment explaining what situations we intend to show the form.